media/js/styleguide/docs/send-to-device.js

```diff
@@ -6,5 +6,10 @@ $(function() {
     'use strict';
 
     var form = new Mozilla.SendToDevice();
+
+    form.geoCallback = function(countryCode) {
+        alert('The country code is: ' + countryCode);
```

Comment:
You can add eslint rules to allow specific statements, including `console.log()`, however that causes issues in older browsers (which is why it's set to error by default)